DOC-3512: Notary FAQ bugs and rewrite (#331)
corda/corda-docs-portal: content/en/platform/corda/4.8/enterprise/notary/faq/notary-failover.md

> # Notary Failover

Reviewer: ```suggestion
# Notary failover
```
Not sure why "Failover" is capitalized. Is "Notary Failover" a proper name of a feature? If no, change to "Notary failover".

Corda has built-in features to allow for the retrying of flows in specific cases of failure.

> See [node-flow-hospital](content/en/platform/corda/4.8/enterprise/node/node-flow-hospital.md) for information that applies

Reviewer: ```suggestion
See [Flow Hospital](content/en/platform/corda/4.8/enterprise/node/node-flow-hospital.md) for information that applies
```

to all flows.

> The FinalityFlow contains notary specific logic which can initiate a backpressure-aware sub-flow since minimum platform version 4.

Reviewer: ```suggestion
The FinalityFlow contains notary-specific logic which can initiate a backpressure-aware sub-flow since minimum platform version 4.
```

## What happens on multiple successful responses caused by retrying?

The first successful response will be mapped to the correct flow and proceed, while the second will be discarded, as the flow that it is

> attempting to map to is no longer in existence. A warning will be logged into the console that a response has nothing to go to, but that

Reviewer: ```suggestion
attempting to map to no longer exists. A warning will be logged into the console that a response has nothing to go to, but that
```

> this is expected behaviour.

Reviewer: ```suggestion
is expected behaviour.
```

## Is it possible to receive a success and failure because of retrying?

Notarization requests are idempotent and can be retried, the same request should lead to the same response when retried.

> Previously notarised transactions are saved so that future repeated requests can be answered appropriately.

Reviewer: ```suggestion
Previously notarized transactions are saved so that future repeated requests can be answered appropriately.
```
